shared-data/default-theme/html/jsapi/index.js

```diff
@@ -400,6 +400,7 @@ Mailpile.API = {
 
 
 // JS App Files /=============================================================
+{% include("jsapi/global/polyfills.js") %}
```

Comment:
To be able to use `modern` JavaScript methods on the String.prototype, e.g. `"x".startsWith("y")` or `"x".includes("y")` I included those polyfills for IE browsers which do not support either [startsWith](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes) nor [includes](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes).

Another solution to manually managing those might be to use something like https://github.com/inexorabletash/polyfill